### PR TITLE
Added TiCDC Alert Rules

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Add description of TiCDC alert rules.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/7354 
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by overvenus — APPROVED** [review]
LGTM

File: ticdc/ticdc-alert-rules.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+## `cdc_resolvedts_high_delay`
+
+- Alert rule:
+
+    (time() - ticdc_processor_resolved_ts / 1000) > 300
+
+- Description:
+
+    resolved ts of a replication task is delayed more than 10 minutes.
```
Comment (Liuxiaozhen12) [inline ticdc/ticdc-alert-rules.md:36]:
```suggestion
     The Resolved TS of a replication task is delayed more than 10 minutes.
```

File: ticdc/ticdc-alert-rules.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+### `cdc_sink_execute_duration_time_more_than_10s`
+
+- Alert rule:
+
+    `histogram_quantile(0.9, rate(ticdc_sink_txn_exec_duration_bucket[1m])) > 10`
+
+- Description:
+
+    It takes a replication task more than 10 seconds to  write data to the downstream database.
```
Comment (Liuxiaozhen12) [inline ticdc/ticdc-alert-rules.md:95]:
```suggestion
    It takes a replication task more than 10 seconds to write data to the downstream database.
```

File: ticdc/ticdc-alert-rules.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+
+### `tikv_cdc_min_resolved_ts_no_change_for_1m`
+
+- Alert rule:
+
+    `changes(tikv_cdc_min_resolved_ts[1m]) < 1 and ON (instance) tikv_cdc_region_resolve_status{status="resolved"} > 0`
+
+- Description:
+
+    The minimum resolved ts 1 of TiKV CDC has not advanced for 1 minute.
```
Comment (Liuxiaozhen12) [inline ticdc/ticdc-alert-rules.md:151]:
```suggestion
    The minimum Resolved TS 1 of TiKV CDC has not advanced for 1 minute.
```

**Review by Liuxiaozhen12 — APPROVED** [review]
rest LGTM

**Pull request merged**